tests/integration/web/info/room_test.py

```diff
@@ -81,6 +81,13 @@ def test_when_rack_is_added_then_return_rack_added_event(self, client):
         assert response.status_code == 200
         assert 'room.rack.added' in response.headers['HX-Trigger']
 
+    def test_room_id_with_slash_no_crash(self, client):
```

Comment:
I like the rhyme, but still

```suggestion
    def test_should_add_rack_for_room_id_with_slash(self, client):
```